internal/test_helpers/course_definition.yml

```
@@ -85,6 +85,11 @@ extensions:
 
       Pipelines allow you to connect multiple commands together, so the output of one command becomes the input of the next command.
 
+  - slug: "history"
+    name: "Command History"
```

```suggestion
    name: "History"
```

(Use exact names in doc)